server/graphql/general/src/lib.rs

```diff
@@ -295,6 +299,24 @@ impl GeneralQueries {
             filter,
         )
     }
+
+    pub async fn currency(
+        &self,
+        ctx: &Context<'_>,
+        currency_id: String,
+    ) -> Result<CurrencyResponse> {
+        get_currency(ctx, &currency_id)
+    }
```

Comment:
If we don't need it maybe not add it for now? at least not on the graphql level? (hard to remove api endpoints without breaking potential users...)